app/scripts/models/auth_brokers/fx-fennec-v1.js

```diff
@@ -13,11 +13,7 @@ define([
   'use strict';
 
   var FxFennecV1AuthenticationBroker = FxDesktopV2AuthenticationBroker.extend({
-    type: 'fx-fennec-v1',
-
-    haltAfterResetPasswordConfirmationPoll: false,
```

Comment:
See [this comment](https://github.com/mozilla/fxa-content-server/pull/3095/files#r40082943) for explanation on this.